src/nanovg.c

```diff
@@ -1398,7 +1407,7 @@ static void nvg__flattenPaths(NVGcontext* ctx)
 		}
 
 		// Enforce winding.
-		if (path->count > 2) {
+		if (state->lineStyle > 1 && path->count > 2) {
```

Comment:
Avoid flipping polygon in line style mode because this will flip UVs and cause dashes to jump around.